src/components/Presentation/PresentationHeader/Object/Boundary/index.jsx

```diff
@@ -0,0 +1,68 @@
+import Vertex from "../Vertex";
+
+const vertexSize = 7;
+
+function Boundary(props) {
+  const { boundaryVertices, onVertexDrag } = props;
+
+  const adjustedVertices = boundaryVertices.map((vertex) => ({
+    x: vertex.x - vertexSize / 2,
+    y: vertex.y - vertexSize / 2,
+  }));
+
+  const svgLines = boundaryVertices.map((vertex, index, vertices) => {
+    const nextVertex = vertices[(index + 1) % vertices.length];
+    return (
+      <line
+        // eslint-disable-next-line react/no-array-index-key
+        key={index}
+        x1={vertex.x}
+        y1={vertex.y}
+        x2={nextVertex.x}
+        y2={nextVertex.y}
+        stroke="red"
+        strokeWidth="2"
+      />
+    );
+  });
+
+  const cursorTypes = [
+    "nwse-resize",
+    "ns-resize",
+    "nesw-resize",
+    "ew-resize",
+    "nwse-resize",
+    "ns-resize",
+    "nesw-resize",
+    "ew-resize",
+  ];
+
+  return (
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+    <div onMouseDown={(e) => e.stopPropagation()}>
```

Comment:
```suggestion
    <div onMouseDown={(event) => event.stopPropagation()}>
```
추후에 저희 이벤트 핸들러 이벤트 객체 event로 이름 통일해야 할 거 같습니다 👍